apps/explorer/src/components/TokenDetails.tsx

```diff
@@ -60,32 +69,31 @@ export const TokenDetails = ({
   }, [destAsset?.decimals, transferAssetReleaseAsset]);
 
   const isNativeToken = useMemo(() => {
-    if (transactionStatusResponse?.transferAssetRelease?.denom?.startsWith('0x')) {
-      return isEvmNativeDenom(transactionStatusResponse?.transferAssetRelease?.denom);
-    }
-    if (transferAssetReleaseAsset) {
-      return (transferAssetReleaseAsset.chainId === transferAssetReleaseAsset?.originChainId &&
-        transferAssetReleaseAsset?.denom === transferAssetReleaseAsset?.originDenom);
-    }
-    return destAsset?.originChainId === destAsset?.chainId && destAsset?.originDenom === destAsset?.denom;
-  }, [destAsset?.chainId, destAsset?.denom, destAsset?.originChainId, destAsset?.originDenom, transactionStatusResponse?.transferAssetRelease?.denom, transferAssetReleaseAsset]);
+    return receivedAsset?.description?.includes('native')
+   
+  }, [receivedAsset?.description]);
```

Comment:
not sure about this

Reply:
yeah i have no clue what the logic should be for the isNativeToken tbh